### ISSUE TITLE
Translations needed of new audio description prompts

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#453]
@GERMANOCARELLA @rococode @Egyrs @ylahav @motchie @jannrobert @vierkantemeter @ShadowBB @mgmar22 @JediLin @rscano @brunowelber @frirasyidi  @marstyson 

Thanks for your help with translating Able Player to non-English languages! 

 I've added some options to the Description Preferences dialog, so users can select the Voice, Rate, and Pitch of the synthesized voice that announces audio description text.   There's new text associated with each of these items that needs to be translated. It's very short, so I think it will be a very quick translation job.  If you have a moment, I would very much appreciate your translations of Lines 209 - 225 in the translations files.  The new text is currently only on [the develop branch](https://github.com/ableplayer/ableplayer/tree/develop).   

Let me know if you have any questions. Thanks again!

**Comment by frirasyidi** [comment]
Hi Terrill,

Can you please clarify what "Rate" means? Does it refer to the reading
speed?

Cheers.


*Fri*, Product Designer
***@***.****

**Comment by JediLin** [comment]
I believe that "rate" means the speed of the running speech as in common TTS or screen reader settings.
BTW, I already submit a pull request for zh-tw translation update. :-)

**Comment by frirasyidi** [comment]
Yeah, I suspected the same. I'm waiting for @terrill to confirm before doing a pull request just in case.

**Comment by terrill** [comment]
@frirasyidi yes, that's correct. "Rate" is the speed of the speech.

**Comment by rianfloo** [comment]
FR version is here: https://github.com/ableplayer/ableplayer/pull/473

**Comment by terrill** [comment]
We're still needing help with translation of the new text in the Descriptions Preferences dialog.  Below are two lists: Languages that have been updated and languages that have not.  I've also tagged particular people who I hope can help with particular languages, but I welcome help from anyone. If **you** can help, please free to claim a language. Also, this is a living comment: I'll edit the lists as more translations are completed. 

### Languages NOT UPDATED 
- Catalan ([ca.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/ca.js)) @rococode 
- Czech ([cs.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/cs.js)) @dukeluke87 
- Spanish ([es.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/es.js)) @Egyrs 
- Hebrew ([he.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/he.js)) @ylahav 
- Italian ([it.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/it.js)) 
- Japanese ([ja.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/ja.js)) @motchie 
- Norwegian ([nb.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/nb.js)) @jannrobert 
- Dutch ([nl.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/nl.js)) @ShadowBB @vierkantemeter 
- Portuguese - Brazil ([pt-br.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/pt-br.js)) @HeadOnF1re
- Turkish ([tr.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/tr.js)) @marstyson 

### Languages Updated 
- Danish (da.js)
- French (fr.js) 
- Indonesian (id.js)
- Swedish (sv.js) 
- Chinese, Traditional - Taiwan (zh-tw.js)

**Comment by ylahav** [comment]
I added the Hebrew missing translations

**Comment by terrill** [comment]
@Head0nF1re thank you for the Portuguese translation! Do you mind submitting an update to the Brazilian Portuguese translation as well ([pt-br.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/pt-br.js))?  It just needs an update to the text in the Audio Description Preferences dialog (currently in English). I could just copy those items over from your Portuguese translation but I suspect there may be slight differences.

**Comment by terrill** [comment]
@ylahav I'm not seeing an updated Hebrew translation anywhere. Did you submit a pull request?  Thank you!

**Comment by ylahav** [comment]
No - I edited the file directly on github - will check it tomorrow

**Comment by Head0nF1re** [comment]
Done, according to what I found online, in that case it's the same.


Yair Lahav ***@***.***> escreveu no dia quinta, 14/07/2022
à(s) 19:09:

**Comment by motchie** [comment]
@terrill I'm sorry for late work. done. Please refer to https://github.com/ableplayer/ableplayer/pull/544 .
